event/http.go

```diff
@@ -10,3 +12,14 @@ type HTTPEvent struct {
 	Method  string              `json:"method"`
 	Params  map[string]string   `json:"params"`
 }
+
+func NewHTTPEvent(r *http.Request, eventData interface{}, headers map[string]string) *HTTPEvent {
```

Comment:
headers are in http.Request, no need to pass them as separate arg.

Reply:
The headers in `http.Request` are in `http.Header` type, these are in transformed `map` format.